architecture/core_concepts/routes.adoc

```diff
@@ -830,6 +831,52 @@ Setting a server-side timeout value for passthrough routes too low can cause
 WebSocket connections to timeout frequently on that route.
 ====
 
+[[whitelist]]
+== Route Specific IP Whitelists
+You can restrict access to a route to a select set of IP addresses by using the 
+*haproxy.router.openshift.io/ip_whitelist* annotation on the route. The whitelist is a space
+separated list of IP addresses and/or CIDRs for the approved users.  Requests from
+IP addresses that are not in the whitelist are dropped.
+
+When the annotation is present for a route, an acl is set up in the route's haproxy backend with the whitelist.
```

Comment:
Deleted 841. It is a level of detail that won't be interesting to the reader. The discussion is found in the haproxy 1.5 documentation. FYI haproxy includes addess control lists (acl). A list can be created and later used in the backend for different purposes. This PR uses the acl as a source of white list source IP addresses.